docs/specs/servers/verify/verify-server-api.md

```diff
@@ -1,30 +1,169 @@
 # Verify Server API
 
+## Generate Challenge (iOS only)
+
+Generates a unique one-time challenge to be later used with other requests as a mean to prevent replay attacks.
+
+### Request
+
+`POST /challenge`
+
+### Responses
+
+- `201` - Challenge generated
+
+```jsonc
+{
+    "id": string
+}
+```
+
+## Register Attestor (iOS only)
+
+Registers an instance of iOS dApp.
+
+["Verify the Attestation"](https://developer.apple.com/documentation/devicecheck/validating_apps_that_connect_to_your_server#3576643) step from Apple docs.
+
+### Request
+
+`POST /attestor/:id`
+```jsonc
+{
+    "attestationObject": [number] 
+}
+```
+- `attestationObject` - byte array representing CBOR encoded ["Attestation Object"](https://developer.apple.com/documentation/devicecheck/validating_apps_that_connect_to_your_server#3576643)
+
+### Responses
+
+- `201` - Attestor registered
+- `400` - Invalid request body or `attestationObject` format
+- `401` - Failed to verify the `attestationObject` according to ["Verify the Attestation"](https://developer.apple.com/documentation/devicecheck/validating_apps_that_connect_to_your_server#3576643)
+- `403` - App ID is not registered on [WalletConnect Cloud](https://cloud.walletconnect.com)
```

Comment:
@Cali93 can you add specs for the Explorer endpoint, where dapp's metadata URL is returned based on the app_id?